iXBRLViewerPlugin/viewer/src/html/fact-details.html

```diff
@@ -15,34 +15,34 @@
 -->
```

Comment:
(Arbitrary line.)

> some ESEF filings have taken to including numeric values twice: once, untagged, with additional elements for styling, and again inside a display: none span, so that it can be tagged.

This seems like a violation of [IXBRL 9.1.2](http://www.xbrl.org/specification/inlinexbrl-part1/rec-2013-11-18/inlinexbrl-part1-rec-2013-11-18.html#d1e4467).  I assume that rule exists for the same reason that motivates this PR: the facts aren't explicitly annotated, so viewer software has no good way of categorizing the facts without a CSS engine (e.g., if we wanted to shift analysis to the plugin to reduce startup cost).  Should we work with the vendor to fix?

Reply:
I think you're right.  I thought there was something in the spec about this, but I failed to find it despite looking several times!

We've already raised this with the vendor, and with ESMA.  I think the issue behind this is an ESEF filing rule about not putting values in `ix:hidden` that are "eligible for transformation".  The ESEF rules don't define what that means, but I think there's an equivalent SEC rule which effectively says that monetary facts are eligible for transformation and thus can't go in `ix:hidden`.  Not withstanding the above rule that makes this technically invalid, using `ix:hidden` with `-esef-ix-hidden` is clearly preferable to using a repeated value with `display: none`.

I should have mentioned that #219 made it so that the viewer will scroll to where the hidden element is, but obviously nothing is highlighted.

Reply:
It's somewhat unfortunate that we're adding code that implicitly condones vendors using CSS tricks to replace a validatable ESEF/SEC errors with a non-validatable iXBRL constraints.  I would personally be inclined to do nothing at all.  If such filings are difficult to view, well, that's exactly why those constraints exist :-).

Reply:
@paulwarren-wk I agree with @brettkail-wk sentiment here but can be persuaded. Do we want to move forward this change given that coding for CSS tricks seems dicey?

Reply:
@derekgengenbacher-wf Let's move forward with this change given the volume of impacted filings.